### ISSUE TITLE
uninitialized constant AllureRSpec::Formatter::Pathname (NameError)

### ISSUE THREAD

**Issue by wuhuizuo** [allure-framework/allure-rspec#70]
## Env:

``` bash
wuhuizuo@debian-wuhuizuo:~/apps/j2test$ ruby -v
ruby 2.1.5p273 (2014-11-13) [x86_64-linux-gnu]
wuhuizuo@debian-wuhuizuo:~/apps/j2test$ gem list allure

*** LOCAL GEMS ***

allure-rspec (0.7.7)
allure-ruby-adaptor-api (0.6.10)
wuhuizuo@debian-wuhuizuo:~/apps/j2test$ gem list rspec

*** LOCAL GEMS ***

allure-rspec (0.7.7)
rspec (3.4.0)
rspec-core (3.4.3)
rspec-expectations (3.4.0)
rspec-mocks (3.4.1)
rspec-support (3.4.1)
```
## Test script:

``` ruby
# filename spec/hello_spec.rb

require 'rspec'
require 'allure-rspec'
AllureRSpec.configure do |c|
  c.output_dir = File.join(__dir__, '..', 'rspec_reports')
end
RSpec.configure do |config|
  config.include AllureRSpec::Adaptor
end

describe 'hello' do
  it 'allure-rspec' do
    expect(true).to be_truthy
  end
end
```
## Run result:

``` bash
$ rspec spec/hello_spec.rb 
/var/lib/gems/2.1.0/gems/allure-rspec-0.7.7/lib/allure-rspec/formatter.rb:42:in `start': uninitialized constant AllureRSpec::Formatter::Pathname (NameError)
        from /var/lib/gems/2.1.0/gems/rspec-core-3.4.3/lib/rspec/core/reporter.rb:189:in `block in notify'
        from /var/lib/gems/2.1.0/gems/rspec-core-3.4.3/lib/rspec/core/reporter.rb:188:in `each'
        from /var/lib/gems/2.1.0/gems/rspec-core-3.4.3/lib/rspec/core/reporter.rb:188:in `notify'
        from /var/lib/gems/2.1.0/gems/rspec-core-3.4.3/lib/rspec/core/reporter.rb:88:in `start'
        from /var/lib/gems/2.1.0/gems/rspec-core-3.4.3/lib/rspec/core/reporter.rb:75:in `report'
        from /var/lib/gems/2.1.0/gems/rspec-core-3.4.3/lib/rspec/core/runner.rb:117:in `run_specs'
        from /var/lib/gems/2.1.0/gems/rspec-core-3.4.3/lib/rspec/core/runner.rb:93:in `run'
        from /var/lib/gems/2.1.0/gems/rspec-core-3.4.3/lib/rspec/core/runner.rb:78:in `run'
        from /var/lib/gems/2.1.0/gems/rspec-core-3.4.3/lib/rspec/core/runner.rb:45:in `invoke'
        from /var/lib/gems/2.1.0/gems/rspec-core-3.4.3/exe/rspec:4:in `<top (required)>'
        from /usr/local/bin/rspec:23:in `load'
        from /usr/local/bin/rspec:23:in `<main>'
```

**Comment by smecsia** [comment]
@wuhuizuo I could not reproduce your problem on my local using Ruby `2.2.1`, RSpec `3.4.0`, AllureRSpec `0.7.7`. Could you provide Dockerfile, for example, to ensure the problem is not with your local setup?

**Comment by ericreeves** [comment]
It would be fantastic if we could get a new build with this fix!  😄   It's not in 0.7.7.

**Comment by schanjr** [comment]
Hi @smecsia . 
I ran into the same issue with my  
"ruby 2.2.3p173 (2015-08-18 revision 51636) [x64-mingw32]"

In one of your files, you're using a standard ruby library "Pathname".
https://github.com/allure-framework/allure-rspec/blob/master/lib/allure-rspec/formatter.rb#L42 


 I used allure-rspec several times in the past. Today it seemed to have stopped importing that standard library which caused this error. Weird Ruby issue. If I explicitly require this library in my spec_helper.rb, it will fix it. 

@wuhuizuo - should see if this fixes your problem as well.
